bci_tester/data.py

```diff
@@ -1018,14 +1035,20 @@ def create_BCI(
         forwarded_ports=[PortForwarding(container_port=6379)],
     )
     for versions, tag in ((("tumbleweed", "15.6", "15.7"), "8.0"),)
+] + [
+    create_BCI(
+        build_tag="private-registry/harbor-valkey:8.0.2",
```

Comment:
Isn't there a tag that is not so specific? This is going to require a lot of churn :-(

Reply:
there's a latest tag also

Reply:
let's use that please